SPECS/ruby/ruby.spec

```diff
@@ -1,21 +1,84 @@
 # The RubyGems library has to stay out of Ruby directory tree, since the
 # RubyGems should be share by all Ruby implementations.
 %global rubygems_dir  %{_datadir}/rubygems
-%global ruby_libdir   %{_datadir}/%{name}
-
-%global gem_dir %{_libdir}/ruby/gems
-%global rubygems_molinillo_version    0.5.7
-%global rubygems_version              3.1.6
-
-%global rubygems_minitest_version     5.15.0
-%global rubygems_power_assert_version 2.0.1
-%global rubygems_rake_version         13.0.6
-%global rubygems_test_unit_version    3.5.3
+%global gem_dir %{_datadir}/ruby/gems
+%global rubygems_version        3.1.6
+# Add version for default gems from https://stdgems.org/
+%global abbrev_version          0.1.0
+%global base64_version          0.1.1
+%global benchmark_version       0.2.0
+%global cgi_version             0.3.1
+%global csv_version             3.2.2
+%global date_version            3.2.2
+%global delegate_version        0.2.0
+%global did_you_mean_version    1.6.1
+%global digest_version          3.1.0
+%global drb_version	         2.1.0
+%global english_version         0.7.1
+%global erb_version             2.2.3
+%global error_highlight_version	0.3.0
```

Comment:
I think there's still two more tabs in these lines. At least that's what a quick search is telling me + `drb_version` is misaligned.